robocin/robocin.github.io
# Update README, navbar, results and partners

# August 2023 Update

This Pull Request brings the following updates to the website:
- **``Update README.md command:``** Python has updated and now the command to start the local host is `Python -m http.server`. The `README.md` file now includes the new version of the command.
- **``Remove 'Members' option in the navbar:``** we decided to remove this option in order to make the Members page innaccessible. The file `membros.html` stills exists (just in case we decide to update it in the future), but users can no longer easily access it through the navbar. They can still access it directly through the link, but they will be met by a 404 page in that case.
- **``Update results:``** competition results were updated in each category page.
- **``Partners:``** updated 'Our Partners' section in each page to display our current partners/sponsors.

### Testing

1. Start the local server by doing `Python -m http.server`
2. Check if the 'Members' option **does not** appear in the navbar (check all pages)
3. Check if competition results for the categories of SSL, VSS and 2D are up to date
4. Check if our partners in the 'Our Partners' section is up to date (check all pages)
    - Our current partners are: Moura/ITEMM, Cesar, Microsoft, Incognia, Neurotech, HSBS, Embraer, INES (but CIn should also be included)

### Notes
- I couldn't find sufficient data for the Papers page. I suggest this is updated in a future issue just so more urgent updates are not kept on hold because of this.
- If you're not seeing any changes on the website, it might be due to outdated cached data. Try accessing it using Incognito Mode, another browser user or browser. Cleaning the browser's cache would also work, but you probably don't want to do that unless you don't mind being logged out of every site you access using the browser.
